### ISSUE TITLE
Issue with notion link

### ISSUE THREAD

**Issue by DobrevDeyan** [kyswtn/vscode-notion#4]
Using the extension, when i copy a link for a page i get "Notion page not found "15999ce0364080ad85d3d1******"". Same page i have changed to be available for anyone with the link from the options in the notion app ?

**Comment by kyswtn** [comment]
Hello, do you mind sharing the link or page ID if it's public? That way I'll be able to debug the behavior myself.

**Comment by DobrevDeyan** [comment]
i have just started using the app and have noticed that it is under private section in the menu, could not find if that is the reason why is this issue... as a document which is not in a workspace for anyone else but me.  Tried to move it to non private somehow but could not. / https://www.notion.so/ICARD-15999ce0364080ad85d3d1cff37d1c5a?pvs=4

**Comment by manuelbds** [comment]
Same issue here. I'm using a private notion page and can't get to integrate with vscode

**Comment by schaubli** [comment]
As shown in the [changelog](https://marketplace.visualstudio.com/items/frenco.vscode-notion/changelog), private pages are no longer supported unfortunately. Only publicly accessible (deployed) Notion pages can be used. As a reference, the Notion page for the YouTube show "Hot Ones" loads correctly for me: https://bensomething.notion.site/Hot-Ones-8bc5367e6c9d4b52a1c2b6595cfd8181

**Comment by kyswtn** [comment]
Yes, that is right. Private page support that was previously implemented was very buggy so I decided to just remove it. It's written out in README as well.
